updater.go

```diff
@@ -188,7 +188,12 @@ func (u Updater) SetWebhook(path string, webhook Webhook) (bool, error) {
 	}
 
 	v := url.Values{}
-	v.Add("url", webhook.URL+"/"+path)
+	if webhook.URL[len(webhook.URL)-1:] != "/" {
```

Comment:
Recommend using `strings.HasSuffix()` instead of this

Reply:
Got it.